advanced/am-pattern-3/values.yaml

```diff
@@ -151,6 +151,20 @@ wso2:
 
       # API Manager's Control Plane specific configurations
       cp:
+        db: 
```

Comment:
Shall we move this out of the cp level and have it in the am level, since db is common to all profiles? Do this for both patterns

Reply:
As we discussed, there is a possibility that CP's DB configuration can differ from GW, and TM servers, Hence keeping the code as it is.